### PR TITLE
ucm2: add support to for Qualcomm RB5 Platform

### PR DESCRIPTION
**Pull request by lumag**

### CODE REVIEW

**Comment by lumag** [comment]
This is the replacement for #63

**Comment by lumag** [comment]
@perexg Any updates/review on this PR?

File: ucm2/codecs/lpass/va-macro/DMIC0DisableSeq.conf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+DisableSequence [
+	cset "name='VA DMIC MUX0' ZERO"
+	cset "name='VA_DEC0 Volume' 0"
+	cset "name='VA_AIF1_CAP Mixer DEC0' 0"
+]
```
Comment (perexg) [inline ucm2/codecs/lpass/va-macro/DMIC0DisableSeq.conf:5]: The lpass directory is not so obvious for other people. Can you rename this to qcom-lpass or so?  Thank you.

Comment (lumag) [reply ucm2/codecs/lpass/va-macro/DMIC0DisableSeq.conf]: @perexg done

**Comment by lumag** [comment]
@perexg Interesting. With this pull request I see the following errors, which are completely unrelated:
```
ERR: bytcr-rt5640/bytcr-rt5640.conf@HiFi.conf: 'If'.'0'.'False'.'Include'.'long'.'If'.'mono'.'True'.'Include'.'mspk'.'SectionDevice'.'Speaker'.'Value'.'If'.'MonoSpkAif1'.'Condition' - False block not executed
```

**Comment by perexg** [comment]
Missing conf.d files ?

<pre>
WARN: Unable to find UCM configuration for 'configs/Qualcomm/Qualcomm-RB5-WSA8815-Speakers-DMIC0.txt'
WARN:   Path#0: '../../../alsa-ucm-conf/ucm2/conf.d/sm8250/Qualcomm-RB5-WSA8815-Speakers-DMIC0.conf'
WARN:   Path#1: '../../../alsa-ucm-conf/ucm2/conf.d/sm8250/sm8250.conf'
</pre>

**Comment by lumag** [comment]
> Missing conf.d files ?
> 
> WARN: Unable to find UCM configuration for 'configs/Qualcomm/Qualcomm-RB5-WSA8815-Speakers-DMIC0.txt'
> WARN:   Path#0: '../../../alsa-ucm-conf/ucm2/conf.d/sm8250/Qualcomm-RB5-WSA8815-Speakers-DMIC0.conf'
> WARN:   Path#1: '../../../alsa-ucm-conf/ucm2/conf.d/sm8250/sm8250.conf'

@perexg Oops. Should be fixed now.

**Comment by perexg** [comment]
Merged. Thank you. If you're using PA, you may consider to add support for the hardware volume (I'm seeing the static setup only).

**Pull request closed without merging**